examples/worlds/gimbal.sdf

```diff
@@ -0,0 +1,390 @@
+<?xml version="1.0" ?>
+<!--
+  Ignition gazebo demo using IMU sensor for Gimbal
+-->
+<sdf version="1.6">
+  <world name="gimbal_system">
+    <physics name="1ms" type="ignored">
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>1.0</real_time_factor>
+    </physics>
+    <plugin
+      filename="ignition-gazebo-physics-system"
+      name="ignition::gazebo::systems::Physics">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-air-pressure-system"
+      name="ignition::gazebo::systems::AirPressure">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-altimeter-system"
+      name="ignition::gazebo::systems::Altimeter">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-imu-system"
+      name="ignition::gazebo::systems::Imu">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-magnetometer-system"
+      name="ignition::gazebo::systems::Magnetometer">
+    </plugin>
```

Comment:
Do we need all of these systems here just to demo the gimbal? The example should show users the minimal way of using that features, so they can take it and incorporate into their projects without bringing unnecessary things.